Cinema/src/main/java/Cinema/Cinema/Main.java

```diff
@@ -0,0 +1,24 @@
+package Cinema.Cinema;
+
+import Cinema.Cinema.models.Movie;
+import Cinema.Cinema.services.MovieService;
+
+public class Main {
+
+	public static void main(String[] args) {
+		MovieService service = new MovieService();
+		
+		service.addMovie(new Movie("Movie", "Documenatry", "U"));

```

Comment:
Might want to put test data into a method so you can easily populate service when you need it for testing- David HL